### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

**Comment by aayushjain-code** [comment]
Go to Wishlist for checking the login signup feature.......
Live Link:   https://lively-kelpie-691d1d.netlify.app/

File: src/context/themeContext.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const ThemeProvider = ({ children }) => {
+  const localStorageTheme = localStorage.getItem("data-theme");
+  const [theme, setTheme] = useState(localStorageTheme ?? "light");
+
+  const toggleLightDarkTheme = () => {
+    setTheme((theme) => (theme === "light" ? "dark" : "light"));
+  };
+
+  useEffect(() => {
+    // document.documentElement.setAttribute("data-theme", theme);
```
Comment (MuskanChhatrasal) [inline src/context/themeContext.js:14]: Remove commented code

**Review by MuskanChhatrasal — APPROVED** [review]
Great UI with clean and readable code. Nothing much to suggest just remove commented code else everything looks good to merge

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything seems good to me just make UI/UX better.
Because I was unable to access the buttons for log in and sign up
Else code is good to merge

**Comment by aayushjain-code** [comment]
Made Few changes Thankyou Merging branches!

**Pull request merged**